python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -23,6 +24,11 @@
     Type,
 )
 
+import numpy as np
+
+from cudf.testing._utils import assert_eq
```

Comment:
This should probably use relative imports like the other imports just below. You'll have to isort these after this change.
```suggestion
from ..testing._utils import assert_eq
```